From a869f8c7e3a9a4f457509636d34cd017192c35a6 Mon Sep 17 00:00:00 2001
From: Tatsuo ONISHI <onishi@latmos.ipsl.fr>
Date: Fri, 9 Dec 2022 18:11:42 +0100
Subject: [PATCH 52/70] (Update) Add a new DMS emission scheme

TYPE : Update

Description:
       Update to the last commit (forgotten modification)
       Namelist option "dmsemis_opt" in Registry/registry.chem are redifined as follows.
       OLD :
          package   dmsmosaic        dmsemis_opt==2               -              -
          package   dms_nightingale  dmsemis_opt==3               -             -
       NEW :
          package   dmsmosaic        dmsemis_opt==3               -              -
          package   dms_nightingale  dmsemis_opt==2               -             -

List of modified files:
   * Registry/registry.chem
---
 Registry/registry.chem | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Registry/registry.chem b/Registry/registry.chem
index b8d94c9..ef3daf4 100644
--- a/Registry/registry.chem
+++ b/Registry/registry.chem
@@ -4415,8 +4415,8 @@ package   seasgocart    seas_opt==1                  -              emis_seas:es
 package   seasmosaic    seas_opt==3                  -              emis_seas2:eseasj,eseasc
 package   dmsgocart    dmsemis_opt==1                  -              -
 #lm: add a dms emission option for cbmz_mosaic and saprc_mosaic
-package   dmsmosaic        dmsemis_opt==2               -              -
-package   dms_nightingale  dmsemis_opt==3               -             -
+package   dmsmosaic        dmsemis_opt==3               -              -
+package   dms_nightingale  dmsemis_opt==2               -             -
 package   volume_approx    aer_op_opt==1                -             -
 package   maxwell_approx   aer_op_opt==2                -             -
 package   volume_exact     aer_op_opt==3                -             -
-- 
2.31.1

